custom/aaa/models.py

```diff
@@ -1193,6 +1195,35 @@ class AggregationInformation(models.Model):
     aggregation_window_end = models.DateTimeField()
 
 
+class AaaFile(models.Model):
```

Comment:
What does this class add over calling `get_blob_db().put()` directly? It doesn't appear that we're trying to cache the downloads in anyway so I think directly calling it without addinga model is a better choice